docs/getting_started_linux_cmake.md

```diff
@@ -0,0 +1,99 @@
+# Getting Started on Linux with CMake
+
+## Prerequisites
+
+### Install CMake
+
+Install CMake version >= 3.13:
+
+```shell
+$ sudo apt install cmake
+```
+
+> Tip:<br>
+> &nbsp;&nbsp;&nbsp;&nbsp;Your editor of choice likely has plugins for CMake,
+such as the Visual Studio Code
+[CMake Tools](https://github.com/microsoft/vscode-cmake-tools) extension.
+
+### Install a Compiler
+
+We recommend Clang. GCC is not fully supported.
+
+```shell
+$ sudo apt install clang
+```
+
+Verify the version with `clang++ --version`. We have tested with the following
+versions:
+
+*   6.0.1
+*   8.0.1
+
+There have been issues with the following versions:
+
+*   [9.0.0-2](https://github.com/google/iree/issues/1216)
+
+Set environment variables:
+
+```shell
+export CC=clang
+export CXX=clang++
+```
+
+## Clone and Build
+
+### Clone
+
+Clone the repository and initialize its submodules:
+
+```shell
+$ git clone https://github.com/google/iree.git
+$ cd iree
+$ git submodule update --init
+```
+
+> Tip:<br>
+> &nbsp;&nbsp;&nbsp;&nbsp;Editors and other programs can also clone the
+repository, just make sure that they initialize the submodules.
+
+### Build
+
+Configure:
+
+```shell
+$ cmake -B build/ .
+```
+
```

Comment:
Put a tip here to link master `CMakeLists.txt` for more configuration controls?

Reply:
Good idea, done.

I'm trying to set a high bar for what instructions stay in this doc (I'm really on the fence about the "compatible" clang versions section...). I also added some comments to the top of these docs as guidance for others editing them in the future.